### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by jaygel179**
- update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
 # racsplit
 Splitter for ractive.js component files
+
```
Comment (dashmug) [inline README.md]: Most I've seen in GitHub readmes use `=============` for h1 and `------------------` for h2.

It looks good as plaintext that way.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+## Usage
+`$ racsplit [-t][--type] <type> [-p][--path] <path/to/file.html>`
+
+
+types: `css | script | modules | template | imports`
+
+
+## Sample
+`$ racsplit -t script -p component/sample-compoenent.html`
```
Comment (mknecht) [inline README.md]: typo

**Comment by mknecht** [comment]
@jaygel179 I'd do two PRs, one for each issue (readme, argument parsing). Since it's open-source you should demonstrate even *more* maturity, not less. :)

**Comment by mknecht** [comment]
actually, three, see your bullet points :)

**Comment by mknecht** [comment]
your project, though. :+1:

**Pull request merged**